docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -87,6 +87,38 @@ To change alert statuses, either:
 * In the Alerts table, select all the alerts you want to change, and then select
 *Take action* -> *Close selected*, *Open selected*, or *Mark in progress*.
 
+[float]
+[[signals-to-cases]]
+=== Add alerts to cases
+From the Alerts table, you can attach one or more alerts to a case by using the *Add to case* button. From here, you can choose to <<signals-to-new-cases, add the alert to a new case>> or <<signals-to-existing-cases, attach it to an existing one>>. There is no limit to the number of alerts that you can add and you can choose alerts from any rule type - custom or pre-built. If the case has been configured to sync its status with associated alerts, the alert's status will update after it's been added. If this setting isn't enabled, the alert's status will not change after you've added it.
```

Comment:
> If the case has been configured to sync its status with associated alerts, the alert's status will update after it's been added. If this setting isn't enabled, the alert's status will not change after you've added it.

Not sure if it's necessary but should we add a sentence about when the `status` would be updated? Like:
`If the case has been configured to sync it's status with associated alerts, the alert's status will update whenever the case's status is changed`

That might be redundant though.

Reply:
Yes, I think it should be clear that the alert's status will change when you change the case's status and sync is on.